src/frontend/shortlist/src/views/VerifiedView.vue

```diff
@@ -0,0 +1,59 @@
+<script>
+import axios from "axios";
+export default {
+  data() {
+    return {
+      isVerified: false,
+      email: "",
+      errorMessage: "",
+    };
+  },
+  created() {
+    //fetching information from url sent to the email address
+    let urlParams = new URLSearchParams(window.location.search);
+    this.isVerified = console.log(urlParams.get("token-valid"));
```

Comment:
The email I got from the system looked like:
http://www.shortlist.nyc/verifyEmail?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNjY3OTQyNzUyLCJpYXQiOjE2Njc5NDI2OTIsImp0aSI6ImJhN2ZhOTZiNjYwOTQ5ZDQ5YTU3YzFiMWVmYTVlYzA5IiwidXNlcl9pZCI6N30.5PpyT4_id5qmS6FLdI3RFXg_UR0iW3RoBXCMLilZjDQ

There is only one URL parameter ("token")

First, I don't know if using window.location.search is the best way to handle getting that value; Please use this.$route.query, or the useRoute composition to get access to the query parameters directly

Second, in order to get isVerified, that needs to be an api call; I would prefer if all api calls (and axios imports) were defined inside of the shortlist.js object so we know where they all are.  On that topic, do we need to have the user re-enter their email address so we can verify that they have both combinations?  (Are there any testing party implications there in that flow?);

Same things goes for the resend link

Reply:
Alex os currently editing the link to display the info, idk how route query is done, but will check out tomorrow.
I will see if is verified is changed into an api call, guess that means a get endpoint needs to be set up, ill talk to Alex about that

Reply:
Yup, I expect to merge it first, so css can be updated and wait for backend, but ill see what I can add as js files